src/assets/src/services/api.ts

```diff
@@ -238,3 +238,12 @@ export const changeMeetingType = async (meeting_id: number, backend_type: string
     await handleErrors(resp);
     return await resp.json() as Meeting;
 }
+
+export const startMeeting = async (meeting_id: number) => {
```

Comment:
`api.ts` uses `snake_case` across the board in its props. It would make a bit more sense for them to be `camelCase` (thus hiding the casing style difference between the JSON schema & TS code in the private implementation of `api.ts`), but if we're going to do that, it should all be done at once.

Reply:
Yeah, we can wait on this, I just saw `snake_case` (for something that wasn't a data structure from the backend) and made a comment.